### DISCUSSION TITLE
insert data in database

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46261] [Q&A]
Hello  everyone i am using a symfony app to display data from server and this data its not fix its change ana i want to create like history for this data in time how can do that ?

**Comment by alessandro-podo** [comment]
use doctrine to insert it into your database

**Reply by Elmatbaisaad** [reply]
how ?? can you tell me more please.

**Reply by MaximePinot** [reply]
[Databases and the Doctrine ORM](https://symfony.com/doc/current/doctrine.html)
